main/php.h

```diff
@@ -434,5 +434,7 @@ END_EXTERN_C()
 typedef bool zend_bool;
 typedef intptr_t zend_intptr_t;
 typedef uintptr_t zend_uintptr_t;
+typedef uint32_t php_uint32;
+typedef int32_t php_int32;
```

Comment:
I would at least move the deprecation comment as well, and given that these typedefs are [soft-deprecated for more than 8 years](fbe50b15dd46ad44d67472e7ba6f12c9015878d1), it *might* make sense to remove them right away. According to a [GH search](https://github.com/search?q=org%3Aphp%20php_uint32&type=code) the only affected PECL extension hosted by the php organization would be APC which is likely no longer working anyway (and might be affected by only moving the typedefs).

Reply:
Oh I thought those were deprecated at the same time, I think removing them straight away makes more sense.